docs/v3/config.rb

```diff
@@ -15,21 +15,25 @@
 set :images_dir, 'images'
 set :fonts_dir, 'fonts'
 
+ignore 'includes/api_resources/examples/*.json'
+
 # Activate the syntax highlighter
 activate :syntax
 
+activate :sprockets
+
+activate :autoprefixer do |config|
+  config.browsers = ['last 2 version', 'Firefox ESR']
```

Comment:
What's this do.

Reply:
activates the sprockets

Reply:
and autoprefixer will ensure your CSS is compatible with older browsers (or whichever browsers you configure here). See https://autoprefixer.github.io/

Reply:
Sprockets is a ruby asset pipeline. It compiles all the js-like and css-like stuff and shoves it into your html in a suitable manor. I think it was always used behind the scenes in middleman, but in 4+ you need to manually activate it or the docs lose style and scripts.

The autoprefixer blocked moved from the :build block below. I'm assuming it adds all the needed browser-specific css prefixes.

Reply:
I mean specifically the `'last 2 version'` 😛

Reply:
For those following along at home, see https://github.com/browserslist/browserslist – this configuration syntax is shared by autoprefixer and lots of front-end-y tools